packages/docusaurus-theme-classic/codeTranslations/ru.json

```diff
@@ -10,7 +10,9 @@
   "theme.blog.post.paginator.newerPost": "Следующий пост",
   "theme.blog.post.paginator.olderPost": "Предыдущий пост",
   "theme.blog.post.readMore": "Читать дальше",
-  "theme.blog.tags": "Теги:",
+  "theme.tags.tagsPageLink": "Просмотреть все теги",
```

Comment:
@lex111 going to merge now but feel free to edit if google translate failed for this string ;)

Reply:
thanks :)